test/e2e/query_test.go

```diff
@@ -289,12 +289,12 @@ func TestQueryLabelNames(t *testing.T) {
 
 	// Outside time range.
 	labelNames(t, ctx, q.HTTPEndpoint(), nil, timestamp.FromTime(now.Add(-24*time.Hour)), timestamp.FromTime(now.Add(-23*time.Hour)), func(res []string) bool {
-		return len(res) == 0
+		return len(res) == 2
 	})
 
 	labelNames(t, ctx, q.HTTPEndpoint(), []storepb.LabelMatcher{{Type: storepb.LabelMatcher_EQ, Name: "__name__", Value: "up"}},
 		timestamp.FromTime(now.Add(-time.Hour)), timestamp.FromTime(now.Add(time.Hour)), func(res []string) bool {
-			// Expected result: [__name__, instance, job, prometheus, replica]
+			// Expected result: [__name__, instance, job, prometheus, receive, replica, tenant_id]
```

Comment:
I like how you updated the commented code to make it work if someone uncomments 😄 😄 
Can we delete this or actually uncomment? 🤗

Reply:
Is it a problem if someone uncomments this 🤔  The E2E test cannot be compiled so we can find the change.